docker/recorder/src/test/java/org/arquillian/cube/docker/impl/client/reporter/TakeDockerEnvironmentTest.java

```diff
@@ -131,4 +165,48 @@ public void should_report_schema_of_docker_composition() {
         assertThat(screenshotEntry.getLink()).isEqualTo("reports/schemas/docker_composition.png");
     }
 
+   @Test
+    public void should_report_log_file(){
+        final TakeDockerEnvironment takeDockerEnvironment = new TakeDockerEnvironment();
+        takeDockerEnvironment.propertyReportEvent = propertyReportEvent;
+        takeDockerEnvironment.reportContainerLogs(new BeforeStop(CUBE_ID), dockerClientExecutor, new ReporterConfiguration());
+        verify(propertyReportEvent).fire(propertyReportEventArgumentCaptor.capture());
+
+        final PropertyReportEvent propertyReportEvent = propertyReportEventArgumentCaptor.getValue();
+        final PropertyEntry propertyEntry = propertyReportEvent.getPropertyEntry();
+        assertThat(propertyEntry).isInstanceOf(FileEntry.class);
+
+        FileEntry fileEntry = (FileEntry) propertyEntry;
+        assertThat(fileEntry.getPath()).isEqualTo("target/reports/logs/tomcat.log");
+    }
+
+    @Test
```

Comment:
I would go with the smallest usable bits of data for the tests, so the narrowed object sounds good to me, as this would be the only thing we are testing. Is it done already?